### PR TITLE
[DOCS] Update SAML URLs

### PR DESCRIPTION
**Pull request by lockewritesdocs**
Updates SAML URLs from `/api/security/v1/saml` to `/api/security/saml/callback`. This change was introduced to the documentation `master` branch in #47999, but was never backported to `7.x` branches.

Preview link: https://elasticsearch_77038.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/7.x/saml-guide-stack.html

Relates to [#81733](https://github.com/elastic/kibana/issues/81733) in the Kibana repo

cc: @kcm

### CODE REVIEW

**Comment by lockewritesdocs** [comment]
@elasticmachine update branch

**Review by azasypkin — APPROVED** [review]
LGTM, thanks! I couldn't remember why we decided not to backport this change, but I can't think of any reason right now.

**Comment by lockewritesdocs** [comment]
@elasticmachine update branch

**Comment by lockewritesdocs** [comment]
The fips failures are not related to changes in this PR. Leaving links to both build scans to trace back:
* part-1-fips: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+pull-request+part-1-fips/376/
* part-2-fips: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+pull-request+part-2-fips/378/

File: x-pack/docs/en/rest-api/security/saml-prepare-authentication-api.asciidoc [hunk 1]
```diff
@@ -77,17 +77,19 @@ POST /_security/saml/prepare
   "realm" : "saml1"
 }
 --------------------------------------------------
+// TEST[skip:Additional configuration needed in yml files]
```
Comment (jkakavas) [inline x-pack/docs/en/rest-api/security/saml-prepare-authentication-api.asciidoc:80]: Lets not skip tests without a valid reason. All we need to do here is to change 

```
  setting 'xpack.security.authc.realms.saml.saml1.sp.acs', 'https://kibana.org/api/security/v1/saml'
```

to 

```
  setting 'xpack.security.authc.realms.saml.saml1.sp.acs', 'https://kibana.org/api/security/saml/callback'
```

in x-pack/docs/build.gradle

Comment (lockewritesdocs) [reply x-pack/docs/en/rest-api/security/saml-prepare-authentication-api.asciidoc:80]: @jkakavas, thanks for the pointer! I wasn't sure how to fix the tests that were breaking the build. I'll create a new PR for updating `build.gradle`.

**Pull request merged**